test/entity.js

```diff
@@ -30,7 +30,7 @@ describe('Entity', function() {
 	}
 
 	it('should auto-instantiate', function() {
-		expect(Entity(resource)).to.be.an.instanceof(Entity);
+		expect(new Entity(resource)).to.be.an.instanceof(Entity);
```

Comment:
The change to the import and having to actually call the constructor now are the only two changes being proposed here - however, I recognize that they are possibly quite disruptive. (I don't really have a good idea of how much usage this library has at this point.)

Reply:
Looks like around [65 places](https://search.d2l.dev/search?project=Brightspace&project=Brightspace-Configs&project=BrightspaceHypermediaComponents&project=BrightspaceUI&project=BrightspaceUILabs&project=lms&full=%22siren-parser%22&defs=&refs=&path=package.json&hist=&type=&xrd=&nn=6&si=path&si=path) are using it, which is more than I expected.

A lot of them are using it just as a parser, ie:
```javascript
import SirenParse from 'siren-parser';
...
const entity = SirenParse(json);
```

So the switch to having to `new Entity(json)` kind of messes with that flow a bit, you know? I'm feeling like we should keep the default export and find a way to keep the old behaviour as well. Something like this in `index.js` maybe?

```javascript
export Entity from './src/entity.js';
export Action from ...;
...;

export default function SirenParse(json) {
  return new Entity(json);
}
```